src/azure-cli/azure/cli/command_modules/resource/tests/latest/test_resource.py

```diff
@@ -1437,7 +1441,7 @@ def test_feature_list(self):
 
     @AllowLargeResponse(8192)
     def test_feature_unregister(self):
-        self.cmd('feature unregister --namespace Microsoft.Network --name AllowLBPreview', checks=self.check('properties.state', 'Unregistered'))
+        self.cmd('feature unregister --namespace Microsoft.Network --name AllowLBPreview', checks=self.check('properties.state', 'Unregistering'))
```

Comment:
Is it possible that it is finished? Can we check Unregistered or Unregistering?

Reply:
Yes, it is possible~ Actually, I sent an email to the service team to ask the expected status before, but I haven't received any reply yet.
If I receive a reply, I will synchronize the information in this PR.

Reply:
Maybe regular expression works